src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,36 @@
-import React from 'react';
+import './CommentInfo.scss';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    id,
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo" key={id}>
```

Comment:
The key prop should not be used here, as it is only necessary when rendering a list of elements. Remove the key={id} from this line.